examples/example5/test_data.xdmf

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0"?>
+<!DOCTYPE Xdmf SYSTEM "Xdmf.dtd" []>
+<Xdmf Version="3.0" xmlns:xi="http://www.w3.org/2001/XInclude">
+  <Domain>
+    <Grid Name="TimeSeries_f_146" GridType="Collection" CollectionType="Temporal">
+      <Grid Name="mesh" GridType="Uniform">
+        <Topology NumberOfElements="512" TopologyType="Triangle" NodesPerElement="3">
+          <DataItem Dimensions="512 3" NumberType="UInt" Format="HDF">test_data.h5:/Mesh/0/mesh/topology</DataItem>
+        </Topology>
+        <Geometry GeometryType="XYZ">
+          <DataItem Dimensions="289 3" Format="HDF">test_data.h5:/Mesh/0/mesh/geometry</DataItem>
+        </Geometry>
+        <Time Value="0.000000000000000e+00" />
+        <Attribute Name="f_146" AttributeType="Scalar" Center="Node">
+          <DataItem Dimensions="289 1" Format="HDF">test_data.h5:/VisualisationVector/0</DataItem>
```

Comment:
@finsberg @emmetfrancis here we observe that dimensions have two components, i.e. (289, 1), which means that rank = 2.

Reply:
This is fine as it is (X, 1), which is later checked for: https://bitbucket.org/fenics-project/dolfin/src/1c52e837eb54cc34627f90bde254be4aa8a2ae17/dolfin/io/HDF5File.cpp#lines-210:212

Reply:
Not sure how to suppress it, could try to set LogLevel ahead of that function call to critical, and reverse it afterwards

Reply:
@emmetfrancis  It should work with something along the lines of:
```
from fenics import *


set_log_level(40)

warning("HELLO")
set_log_level(10)

warning("Bye")
```
as this yields
```bash
root@dokken-XPS-9320:~/shared# python3 mwe.py 
Authorization required, but no authorization protocol specified
Authorization required, but no authorization protocol specified
*** Warning: Bye
```

Reply:
Thanks!